### PR TITLE
HTTP request complete

### PR DESCRIPTION
**Pull request by ilanabonell**
Made edits to sports-near-me job to add a HTTP get request to Gwinnett stripers schedule. HTML was pulled into the sports-near-me log.

### CODE REVIEW

File: server/src/jobs/job.go [hunk 1]
```diff
@@ -7,8 +7,7 @@ import (
 	"server/src/logger"
 	"server/src/nosql_db"
 	"server/src/sql_db"
-
-	"github.com/gin-gonic/gin"
+	"sync"
```
Comment (edverma) [inline server/src/jobs/job.go:10]: Please group this import with the std lib imports above

File: server/src/jobs/sports-near-me_job.go [hunk 2]
```diff
@@ -3,23 +3,27 @@ package jobs
 import (
 	"context"
 	"fmt"
+	"sync"
 	"time"
 
+	"io"
```
Comment (edverma) [inline server/src/jobs/sports-near-me_job.go:9]: Please group these with the std lib imports above

**Review by edverma — COMMENTED** [review]
Nice description on the PR

File: server/src/jobs/sports-near-me_job.go (hunk 2 above)
Comment (ilanabonell) [reply server/src/jobs/sports-near-me_job.go:6]: complete

File: server/src/jobs/job.go (hunk 1 above)
Comment (ilanabonell) [reply server/src/jobs/job.go:10]: Complete

File: server/src/cache/init.go [hunk 3]
```diff
@@ -33,6 +34,8 @@ func Initialize() *Client {
 		Password: "", // no password set
 		DB:       0,  // use default DB
 	})
+	//connection string: client:= ConnectionMultiplexer.Connect($"{Host_name}:{Port_Number},password={pass}");
```
Comment (edverma) [inline server/src/cache/init.go:37]: Please remove these comments

**Review by edverma — APPROVED** [review]

**Pull request merged**